js/build.js

```diff
@@ -341,6 +339,7 @@ Fliplet.Widget.instance('chart-line-1-1-0', function(data) {
             }
           },
           series: [{
+            name: ' ',
```

Comment:
Removes the default `"Series " + (index + 1)` series name, which isn't translatable.